GameCode

```diff
@@ -1,6 +1,6 @@
 const getUserChoice = userInput => {
     userInput = userInput.toLowerCase();
-    if (userInput === 'rock' || userInput === 'paper' || userInput === 'scissors') {
```

Comment:
This initially confused me because I thought I could do one statement with a bunch of pipes between each possible input. I was a dummy then but I'm smarter now.